torchmdnet/scripts/train.py

```diff
@@ -77,8 +78,6 @@ def get_argparse():
     parser.add_argument('--prior-model', type=str, default=None, help='Which prior model to use. It can be a string, a dict if you want to add arguments for it or a dicts to add more than one prior. e.g. {"Atomref": {"max_z":100}, "Coulomb":{"max_num_neighs"=100, "lower_switch_distance"=4, "upper_switch_distance"=8}', action="extend", nargs="*")
 
     # architectural args
-    parser.add_argument('--charge', type=bool, default=False, help='Model needs a total charge. Set this to True if your dataset contains charges and you want them passed down to the model.')
-    parser.add_argument('--spin', type=bool, default=False, help='Model needs a spin state. Set this to True if your dataset contains spin states and you want them passed down to the model.')
```

Comment:
This could break previous yamls. Maybe it is better to leave them for now, marked as deprecated and document them as "This option is obsolete and ignored".

Reply:
Are you sure about it? In theory the test_examples is using yaml in which `charge` and `spin` are defined but it's not complaining about it because the code it's just skipping and ignoring them.

Reply:
No, you cannot. The test_examples it's just ignoring the check called in `LoadFromFile(argparse.Action)`. I will put them back and marked as deprecated